test/jdk/java/awt/print/PrinterJob/TestMediaTraySelection.java

```diff
@@ -23,7 +23,7 @@
 /*
  * @bug 6357887 8165146
  * @summary  Verifies if selected printertray is used
- * @requires os.family == "linux"
+ * @requires (os.family == "linux" | os.family == "mac")
```

Comment:
You should add this bugid here since your regression testcase is suppose to test the bug

Reply:
Done! Thanks!